jsonschema/golangci.v1.57.jsonschema.json

```
@@ -351,9 +351,9 @@
         "timeout": {
           "description": "Timeout for the analysis.",
           "type": "string",
-          "pattern": "^\\d*[sm]$",
+          "pattern": "^(\\d*m)?\\d*[sm]$",
```

```suggestion
          "pattern": "^(\\d*m)?\\d+[sm]$",
```
Shouldn't this at least require 1 digit? 🤔